models/comment-model.js

```diff
@@ -19,3 +19,29 @@ ORDER BY created_at DESC;`,
       return rows;
     });
 };
+exports.inserCommentsForArticleByArticleId = (article_id, username, body) => {

```

Comment:
typo - but up to you if you want to name it like this

Reply:
Done.